embedded-graphics/src/image/image_raw.rs

```diff
@@ -62,7 +62,7 @@ pub type ImageRawBE<'a, C> = ImageRaw<'a, C, BigEndian>;
 /// // raw data gets converted into.
 /// let raw_image: ImageRaw<BinaryColor> = ImageRaw::new(DATA, 12, 5);
 ///
-/// let image = Image::new(&raw_image, Point::zero());
+/// let image: Image<_, BinaryColor> = Image::new(&raw_image, Point::zero());
```

Comment:
I'm not sure about this one. There are 3 possible ways of handling type annotations for images with `ImageRaw` data:
1. Annotate both `Image` and `ImageRaw`
   This is a bit redundant.
2. Only annotate `ImageRaw`
   Not consistent with the BMP and TGA examples, but having the annotation at this point _feels_ right IMO, because it's the place where the format of the stored data is defined.
3. Only annotate `Image`
   Consistent with the BMP and TGA examples.

Any opinions on this or should we just leave it as it is now?